### PR TITLE
Update overview.md

### PR DESCRIPTION
**Pull request by Hitisha-G**
Extended description / introduction of GRPC to be understandable to beginners, exempting the need to go through other sources for basic information.

### CODE REVIEW

File: docs/edge/architecture/overview.md [hunk 1]
```diff
@@ -58,4 +58,5 @@ The JSON RPC layer is an API layer that dApp developers use to interact with the
 The TxPool layer represents the transaction pool, and it is closely linked with other modules in the system, as transactions can be added from multiple entry points.
 
 ## GRPC
```
Comment (DannyS03) [inline docs/edge/architecture/overview.md:60]:
```suggestion
## gRPC
```

File: docs/edge/architecture/overview.md [hunk 2]
```diff
@@ -58,4 +58,5 @@ The JSON RPC layer is an API layer that dApp developers use to interact with the
...
-The GRPC layer is vital for operator interactions. Through it, node operators can easily interact with the client, providing an enjoyable UX.
\ No newline at end of file
+GRPC or Google Remote Procedure Call is a robust open-source RPC framework used to build scalable and fast APIs.It was initially created by Google.
```
Comment (DannyS03) [inline docs/edge/architecture/overview.md:61]:
```suggestion
gRPC, or Google Remote Procedure Call, is a robust open-source RPC framework that Google initially created to build scalable and fast APIs.
```

File: docs/edge/architecture/overview.md [hunk 3]
```diff
@@ -58,4 +58,5 @@ The JSON RPC layer is an API layer that dApp developers use to interact with the
 The TxPool layer represents the transaction pool, and it is closely linked with other modules in the system, as transactions can be added from multiple entry points.
 
 ## GRPC
-The GRPC layer is vital for operator interactions. Through it, node operators can easily interact with the client, providing an enjoyable UX.
\ No newline at end of file
+GRPC or Google Remote Procedure Call is a robust open-source RPC framework used to build scalable and fast APIs.It was initially created by Google. 
+The GRPC layer is vital for operator interactions. Through it, node operators can easily interact with the client, providing an enjoyable UX.
```
Comment (DannyS03) [inline docs/edge/architecture/overview.md:62]:
```suggestion
The gRPC layer is vital for operator interactions. Through it, node operators can easily interact with the client, providing an enjoyable UX.
```

**Review by DannyS03 — CHANGES_REQUESTED** [review]
thanks @Hitisha-G

**Review by ashucoder9 — APPROVED** [review]

**Review by DannyS03 — APPROVED** [review]

**Pull request merged**